### PR TITLE
Add submission: 10L Slide64 Mid6, val_bpb=1.1732

### PR DESCRIPTION
**Pull request by GLDRoger**
This PR submits an updated valid `10-minute / 8xH100 / <16MB` run from the 10-layer sliding-window family.

Result:
- `final_sliding_window_exact val_bpb: 1.17319477`
- `final_quant_zlib_roundtrip_exact val_bpb: 1.20735381`
- `Total submission size quant+zlib: 15846677 bytes`

What changed relative to the earlier valid `mid6` submission in this PR:
- kept the same `10L` architecture
- kept sliding-window eval at `1024/64`
- kept fp16 tied-embedding export
- kept the valid mixed export policy with only middle blocks `3,4,5,6` at int6
- kept the stronger Muon crossover schedule
- changed only `TIED_EMBED_LR` from `0.10` to `0.08`

Why this variant:
- the earlier valid `mid6` run reached `1.17334285` under the cap
- increasing eval context to `2048` did not help and badly regressed the sliding metric
- a smaller tied-embedding LR gave a measurable score improvement while also shrinking the final compressed artifact slightly

Files included in the submission folder:
- `README.md`
- `submission.json`
- `train.log`
- `train_gpt.py`

Quick validation:
```bash
ls records/track_10min_16mb/2026-03-20_10L_Slide64_Mid6
python3 -m py_compile records/track_10min_16mb/2026-03-20_10L_Slide64_Mid6/train_gpt.py
rg -n "Total submission size quant\\+zlib|final_quant_zlib_roundtrip_exact|final_sliding_window_exact" \
  records/track_10min_16mb/2026-03-20_10L_Slide64_Mid6/train.log
cat records/track_10min_16mb/2026-03-20_10L_Slide64_Mid6/submission.json
```

### CODE REVIEW

**Comment by MatoTeziTanka** [comment]
### Community Review — Add submission: 10L Slide64 Mid6, val_bpb=1.1732

**BPB:** 1.1732 | **Compliance:** LOOKS CLEAN — pure-neural submission, no TTT/SLOT/n-gram-cache

**What I found in the code** (head SHA `b413f858fee2`, file `records/track_10min_16mb/2026-03-20_10L_Slide64_Mid6/train_gpt.py`):

Static code review found no TTT adaptation function, no SLOT optimization loop, no n-gram-cache class, and no pre-quant val-token fine-tune. The eval path uses the standard sliding-window stride-64 pattern. The submission is a pure-neural architecture iteration on the standard SP1024/SP4096/SP8192 baseline.

**CPU smoke test (CT2038 proteus-engine, 2026-04-11):** import OK in 0.04s, dim=512, layers=9, vocab=1024, code=65668 B, SMOKE_TEST_PASS

**Verdict:** LOOKS CLEAN.

**Recommendation to @cocohearts @valerio-oai @0hq @yuzhougu-oai @notapplica:** **MERGE** pending the usual record-track checks (3-seed validation, under-16MB artifact cap, ≤600s train + ≤600s eval on 8×H100 SXM). No compliance flags from the classification pass — this looks like a clean pure-neural iteration on the standard baseline.

*Auto-classification caveat:* this review was drafted by the AST-based classifier. If there's a non-standard eval mechanism (logit postprocessing, hedge mixing, etc.) that I missed because it's factored into a helper file or a non-standard function name, please flag it and I'll re-run the audit manually.

---
*Reviewed by [@MatoTeziTanka](https://github.com/MatoTeziTanka) — [The Agora](https://matotezitanka.github.io/parameter-golf). CPU smoke test (CT2038 proteus-engine, 2026-04-11): import OK in 0.04s, dim=512, layers=9, vocab=1024, code=65668 B, SMOKE_TEST_PASS. Classification via deterministic AST-based `classify_prs.py` (pattern bank derived from ~65 manually-reviewed PRs earlier in the 2026-04-11 sweep). This review was auto-drafted from a template and spot-checked before posting — if the template misread your code, please call it out so I can iterate the classifier.*
